stacks/python/2.1.0/devfile.yaml

```diff
@@ -2,17 +2,15 @@ schemaVersion: 2.1.0
 metadata:
   name: python
   displayName: Python
-  description:
-    'Python is an interpreted, object-oriented, high-level programming language with dynamic semantics.
-    Its high-level built in data structures, combined with dynamic typing and dynamic binding, make it very attractive for Rapid Application Development, as well as for use as a scripting or glue language to connect existing components together.'
+  description: Python 3.9.x application with flask microframework and Pip
```

Comment:
I like the original description for the stack description. Maybe just add the version number in